drivers/spi/spi_ll_stm32.c

```diff
@@ -643,6 +643,13 @@ static int transceive(const struct device *dev,
 
 	LL_SPI_Enable(spi);
 
+#if CONFIG_SOC_SERIES_STM32H7X
+	/*
+	 * Add a small delay after enabling to prevent transfer stalling at high
+	 * system clock frequency (see errata sheet ES0392).
+	 */
+	k_busy_wait(1);
```

Comment:
```suggestion
	k_busy_wait(1U * USEC_PER_MSEC);
```

Reply:
Done

Reply:
My bad, this is indeed a wrong understanding from my part.
@gautierg-st Please add and use a define which mention the unit for clarity.

Reply:
Done